### DISCUSSION TITLE
Add More filters

### DISCUSSION THREAD

**Discussion by devyash2930** [devyash2930/ShopSync-SE#4] [Ideas]
By far there is only 1 filter that is the price filter. I think we can add some more filters for the search results

**Comment by vatsal-dp** [comment]
yeah true, we can allow to select more than one website, also we can add sorting functionality by price as well.

**Comment by devyash2930** [comment]
Added the company filters. But I think we should also add the reset button for these filters. Also one thing I noticed is that when new search is done, the previous filters are not cleared from the streamlit state
![Screenshot from 2024-10-26 16-23-26](https://github.com/user-attachments/assets/5ef7e940-061e-4fa3-82d2-52b936a71228)

**Reply by smitraval24** [reply]
Yeah nice idea. We can apply that.

**Comment by devyash2930** [comment]
Added reset and sort filters
![Screenshot from 2024-10-26 18-08-26](https://github.com/user-attachments/assets/652bc4df-fd74-42e2-a973-a5190dc71810)

**Comment by devyash2930** [comment]
Just for reference, this is how to add custom CSS to a button:
https://stackoverflow.com/questions/69478972/how-to-style-a-button-in-streamlit
